### PR TITLE
Update Event “fil-bangkok-2024”

### PR DESCRIPTION
**Pull request by elsanguyen1**
Automatically generated by Decap CMS

### CODE REVIEW

**Review by gmoranxyz — APPROVED** [review]

**Comment by gmoranxyz** [comment]
converted image to webp 200 x 300

File: src/content/events/fil-bangkok-2024.md [hunk 1]
```diff
@@ -23,17 +23,115 @@ luma-events-section:
...
+        - title: FIL Bangkok Co-Working Space
+          description: >-
+            FIL Bangkok Co-Working Space at Gaysorn Urban Resort. FIL Bangkok
+            Co-Working Space is your home base in Bangkok to expand your
+            network, knowledge, and opportunities within the Filecoin ecosystem.
+            From November 6th to 8th, this space is perfect for co-working,
+            meetings, and more!
+
+
+            ​FIL Dev Summit is a gathering of developers, builders, and engaged community members who want to contribute to the core protocol and network evolution of Filecoin. The summit is a place for meaningful and impactful conversations that help push Filecoin forward. (separate registration required).
```
Comment (mirhamasala) [inline src/content/events/fil-bangkok-2024.md:46]:
```suggestion
            ​FIL Dev Summit is a gathering of developers, builders, and engaged community members who want to contribute to the core protocol and network evolution of Filecoin. The summit is a place for meaningful and impactful conversations that help push Filecoin forward (separate registration required). 
```

File: src/content/events/fil-bangkok-2024.md [hunk 2]
```diff
@@ -23,17 +23,115 @@ luma-events-section:
...
```
Comment (mirhamasala) [inline src/content/events/fil-bangkok-2024.md:80]:
```suggestion
            ​FIL Dev Summit is a gathering of developers, builders, and engaged community members who want to contribute to the core protocol and network evolution of Filecoin. The summit is a place for meaningful and impactful conversations that help push Filecoin forward (separate registration required). 
```

File: src/content/events/fil-bangkok-2024.md [hunk 3]
```diff
@@ -23,17 +23,115 @@ luma-events-section:
...
```
Comment (mirhamasala) [inline src/content/events/fil-bangkok-2024.md:113]:
```suggestion
            ​FIL Dev Summit is a gathering of developers, builders, and engaged community members who want to contribute to the core protocol and network evolution of Filecoin. The summit is a place for meaningful and impactful conversations that help push Filecoin forward (separate registration required). 
```

File: src/content/events/fil-bangkok-2024.md [hunk 4]
```diff
@@ -23,17 +23,115 @@ luma-events-section:
...
+        - title: FIL Dev Summit 5
+          description: >-
+            FIL Dev Summit is a gathering of developers, builders, and engaged
+            community members who want to contribute to the core protocol and
+            network evolution of Filecoin (think IPFS Thing, but bigger!). This
+            summit is more than just a meetup—it's a place for meaningful and
+            impactful conversations that help push Filecoin forward.
+
+
+            Find the full agenda at /fil-dev.io/FDS-5
```
Comment (mirhamasala) [inline src/content/events/fil-bangkok-2024.md:130]:
```suggestion
            Find the full agenda at fil-dev.io/FDS-5
```

File: src/content/events/fil-bangkok-2024.md [hunk 5]
```diff
@@ -23,17 +23,115 @@ luma-events-section:
...
```
Comment (mirhamasala) [inline src/content/events/fil-bangkok-2024.md:97]:
```suggestion
            Find the full agenda at fil-dev.io/FDS-5
```

File: src/content/events/fil-bangkok-2024.md [hunk 6]
```diff
@@ -23,17 +23,115 @@ luma-events-section:
...
```
Comment (mirhamasala) [inline src/content/events/fil-bangkok-2024.md:63]:
```suggestion
            Find the full agenda at fil-dev.io/FDS-5
```

**Comment by mirhamasala** [comment]
@gmoranxyz - I left a few commit suggestions regarding copy.

Just checking - But are @elsanguyen1 and you making sure to check the [preview link](https://vercel.live/open-feedback/filecoin-foundation-si-git-c52606-filecoin-foundations-projects.vercel.app?via=pr-comment-visit-preview-link&passThrough=1) before asking for a review?

- Can we ensure we delete unused images? 🙏🏼
- The image that is being used, cuts of Mrs. Callon-Butler's head quite a bit

CleanShot 2024-10-21 at 10 30 03@2x

CleanShot 2024-10-21 at 10 28 51@2x

File: src/content/events/fil-bangkok-2024.md [hunk 7]
```diff
@@ -53,6 +151,12 @@ speakers:
     linkedin: https://www.linkedin.com/in/alexanderkinstler/
     image:
       src: /assets/images/alexander-kinstler-headshot.webp
+  - name: Leah Callon-Butler
+    title: "Director "
```
Comment (mirhamasala) [inline src/content/events/fil-bangkok-2024.md:155]:
```suggestion
    title: "Director"
```

**Comment by mirhamasala** [comment]
@gmoranxyz - Please check the following:

- Previous message regarding speaker image [>>](https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/742#issuecomment-2426123604)
- Are these timing correct? They seem a bit odd? 

CleanShot 2024-10-23 at 19 42 02@2x

**Pull request merged**